woodwork/utils.py

```diff
@@ -411,6 +411,20 @@ def get_valid_spearman_types():
     return valid_types
 
 
+def get_valid_correlation_metrics():
```

Comment:
is this only used in testing / error handling? Also wondering if we should have this tied to a list defined in woodwork instead of hard-coding it.

Reply:
Like should we have a constant list somewhere that defines all of them and pull from that list. So that we don't have to define what these are in multiple places.

Reply:
Moved the list to `config_defaults`!